### PR TITLE
Feature/before asset output callback

### PR DESCRIPTION
**Pull request by SowmediaNL**

### CODE REVIEW

**Review by cascornelissen — CHANGES_REQUESTED** [review]
Sorry for the late review, apparently I forgot to submit it 🤦🏼‍♂️

File: docs/options.md [hunk 1]
```diff
@@ -215,3 +216,10 @@ Name for the SCSS variable that is used for the Sass map containing [user-define
 
 #### `styles.variables.mixin` – `'sprite'`
 Name for the SCSS variable that is used for the Sass mixin.
+
+#### `styles.callback` – `'callback'`
```
[the remark this answers is not in the document]
Comment (SowmediaNL) [reply docs/options.md]: Thanks! The change was comitted.

**Comment by cascornelissen** [comment]
Code LGTM, would you mind squashing your commits into one commit to keep `master` history clean?

**Comment by SowmediaNL** [comment]
How do we squash the commits? Is that not up to the pull-request merger? I have no options for it in Github.

**Comment by cascornelissen** [comment]
If you're not sure how to do it manually (you usually do it locally), I'll create a merge commit instead, no problem 👍🏼

**Review by cascornelissen — APPROVED** [review]

**Comment by cascornelissen** [comment]
This feature was included in release [`3.7.0`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.7.0), thanks for your contribution! 🚀

**Pull request merged**